modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -411,6 +417,32 @@ private void _validateParametersUnicodeProperties(
 				_objectDefinitionPersistence.fetchByPrimaryKey(
 					objectDefinitionId);
 
+			if (parametersUnicodeProperties.containsKey(
```

Comment:
Hey Paulo, I would refactor this code to the following one:
`
			String objectDefinitionExternalReferenceCode = GetterUtil.getString(
					parametersUnicodeProperties.remove("objectDefinitionExternalReferenceCode"));

			if (Validator.isNotNull(objectDefinitionExternalReferenceCode)) {
				ObjectDefinition existingObjectDefinition =
					_objectDefinitionPersistence.fetchByERC_C(
						objectDefinitionExternalReferenceCode, companyId);

				if (!Objects.isNull(existingObjectDefinition)) {
					parametersUnicodeProperties.put(
						"objectDefinitionId",
						String.valueOf(
							objectDefinition.getObjectDefinitionId()));

					objectDefinition = existingObjectDefinition;
				}
			}
`
Do you agree? You can improve if you wanto to :+1: